README-DEV.md

```diff
@@ -13,7 +13,7 @@ This mono-repository contains the TypeScript packages of Mobile Jazz Harmony _fr
 
 Anytime you're working in the project ensure you `nvm use` to stay on the correct NodeJS version.
 
-## Prior to publish a new version
+## Before publishing a new version
```

Comment:
For me it just sounds weird. But I can't give you a proper answer; I'm not sure if it's correct or not.

Reply:
It is not grammatically wrong. It is just more formal. 

It probably sounds similarly formal in Spanish: "Previo a la publicación" vs "Antes de publicar"

Reply:
Exactly, I put it in a formal way. I was asking because it was not suggested as other changes, but changed directly as a commit and I didn't understand why.